nim/tests/test_routes.py

```diff
@@ -0,0 +1,103 @@
+from nim.app import app
+import pytest
+import json
+
+
+@pytest.fixture
+def test_app():
+    return app.test_client()
+
+
+def test_index(test_app):
+    response = test_app.get('/', content_type='html/text')
+    assert response.status_code == 200
+
+
+def test_new_success(test_app):
+    new_game_response = test_app.post(
```

Comment:
`response` should be enough, we'll be using that everywhere so might as well start by being consistent.

Reply:
Fair enough. My intention was to keep it specific as there will be other routes tested in this file, But since it's in the scope of the function it should be cool.
The fact that 'Response' is an existing thing in flask also made me cautious around that name, but I guess the capital should make it obvious enough that they're different.

Reply:
Or `resp` if you prefer.
My reasoning is that we'll have to do quite a few of those tests so if we can make it easier to copy/paste or refactor by keeping the names consistent, then that's probably a good thing to do.

Reply:
Yeah I do think you're right. My above arguments are why I didn't do it in the first place, but after thinking about it I think there is no harm in calling it `response` 🙂 I made this change